### PR TITLE
Added custom Bolt authorization to recognize updated active token gen…

### PR DESCRIPTION
**Pull request by RMK0110**
…eration with login credentials.

### CODE REVIEW

**Comment by hunterowens** [comment]
should we fail if num / trips status_changes == 0, or is it assumed that there we actually _0_ trips for the time range?

**Comment by ian-r-rose** [comment]
I suppose that is up to us. To me, zero trips from the API endpoint is a valid response for the purposes of the DAG, even if it is invalid for the purpose of compliance with MDS.

**Comment by hunterowens** [comment]
reasonable. perhaps some sort of log / email for monitoring purposes.

File: dags/transportation/dockless/dockless_elt.py [hunk 1]
```diff
@@ -105,6 +107,41 @@ def connect_aws_s3():
...
+    <no URL>
+    Currently, your config needs:
+    * email
+    * password
+    * mds_api_url
+    * token_url
+    """
+    def __init__(self, provider):
+        """
+        Acquires the bearer token for Spin before establishing a session.
```
Comment (hunterowens) [inline dags/transportation/dockless/dockless_elt.py]: @RMK0110 can you change this to read _spin_

Comment (hunterowens) [reply dags/transportation/dockless/dockless_elt.py]: (sorry, bolt. This can be done on upstreaming.

**Pull request merged**